frontend/src/metabase/styled-components/containers/GlobalStyles/GlobalStyles.tsx

```diff
@@ -42,6 +42,26 @@ export const GlobalStyles = (): JSX.Element => {
       --mb-color-text-medium: ${color("text-medium")};
       --mb-color-text-white: ${color("text-white")};
       --mb-color-warning: ${color("warning")};
+      --mb-color-border-alpha-30: color-mix(
+        in srgb,
+        var(--mb-color-border) 30%,
+        transparent
+      );
+      --mb-color-text-light-alpha-86: color-mix(
+        in srgb,
+        var(--mb-color-text-white) 86%,
+        transparent
+      );
```

Comment:
nit: Since it might not matter to human eyes, how about changing this value to a more rounded one like 85%? I saw one instance you combined 14% with 15% into a single color which makes sense, since they should be indistinguishable.

Reply:
Good point, will do the same for this one 👍🏻